generators/heroku/index.js

```diff
@@ -163,7 +163,7 @@ module.exports = class extends HerokuGenerator {
 
           this.log(chalk.bold('\nDeploying application'));
 
-          const herokuPush = execCmd('git push heroku HEAD:main', { maxBuffer: 1024 * 10000 });
+          const herokuPush = execCmd('git push heroku HEAD:main');
```

Comment:
Are you sure this is an excess param?  I think I added it because Heroku spits out a ton of logs when building the project.  I'm not positive that's why, would have to generate a project and push to Heroku to be sure.  The default is 1024*1024 which is around 1 Megabyte.

Otherwise everything else looks great, thanks for the help 👍

Reply:
it may be required, however it may have to be supplied within the first param like so:
```suggestion
          const herokuPush = execCmd(`git push heroku HEAD:main { maxBuffer: 1024 * 10000 }`);
```

I removed it because, `util.promisify()` returns a function (`execCmd` in our case) that accepts a single param

I feel I should remove this change for now, investigate more and open another PR if what I think is correct.

Reply:
I'm also curious if you have any thoughts on how we could test this via an integration test

Reply:
It's used the same way in the main generator so I think it's correct, but let me know what you discover.  https://github.com/jhipster/generator-jhipster/blob/e57796b3c2758b5761b8ba8cfa65e582fc8f7449/generators/heroku/index.js#L711

For integration testing this generator, we should add something like the main generator as well: https://github.com/jhipster/generator-jhipster/blob/e57796b3c2758b5761b8ba8cfa65e582fc8f7449/test/heroku.spec.js

Merging, thanks again for the help